UPGRADE.md

```diff
@@ -35,6 +35,20 @@ $ make migrate
       'dal_select2',
   )
   ```
+- `WEB_ANALYTICS_ID` is now a mandatory settings. You can set it to `None` if you do not
```

Comment:
It sucks no? Why not make None a default?

Reply:
Yes, finally it seems to be a bug.
https://github.com/openfun/richie/pull/1624/commits/e13853b0294b2dd0d71a182e056d4b1242f3252e